### PR TITLE
Updates the el-collectooorr to use ACN to share tendermint config at startup

### PR DESCRIPTION
**Pull request by angrybayblade**

### CODE REVIEW

File: temp/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export SERVICE_ELCOLLECTOOORR_RPC_0="http://hardhat:8545"
```
Comment (0xArdi) [inline temp/build.sh:1]: Is this file necessary?

Comment (angrybayblade) [reply temp/build.sh:1]: Nope

**Comment by 0xArdi** [comment]
#170
